WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderMenuAdapter.java

```diff
@@ -72,7 +114,6 @@ public View getView(int position, View convertView, ViewGroup parent) {
         holder.mText.setText(textRes);
         holder.mText.setTextColor(AppCompatResources.getColorStateList(convertView.getContext(), textColorRes));
         ColorUtils.INSTANCE.setImageResourceWithTint(holder.mIcon, iconRes, iconColorRes);
-
         return convertView;
     }
 
```

Comment:
Suggestion (💡): This is not related to your PR but I generally try to remove all warning from a class when working with it. In this case the `ReaderMenuHolder` and `ReaderMenuSpacerHolder` classes can become static. Consider checking the IDE warning and applying the suggested quick fix.